packages/data/addon/adapters/dimensions/bard.ts

```diff
@@ -8,24 +8,39 @@
 import { assert, warn } from '@ember/debug';
 import { inject as service } from '@ember/service';
 import { getOwner } from '@ember/application';
-import { assign } from '@ember/polyfills';
 import EmberObject from '@ember/object';
 import { configHost, getDefaultDataSourceName } from '../../utils/adapter';
 import { serializeFilters } from '../bard-facts';
+import { Filter } from '../fact-interface';
+import { DimensionMetadata } from 'navi-data/models/metadata/dimension';
 
 const SUPPORTED_FILTER_OPERATORS = ['in', 'notin', 'startswith', 'contains'];
 
 /**
  * @enum {String} - mapping of dimension field names to URL dimension field names
  */
-const URL_FIELD_NAMES = {
+const URL_FIELD_NAMES: Dict<string> = {
   description: 'desc'
 };
 
 const SEARCH_TIMEOUT = 30000;
 
 const CLIENT_ID = 'UI';
 
+type DimensionFilter = {
+  field: string;
+  operator: string;
+  values: string[];
+};
```

Comment:
Should we enforce `type: 'dimension'` here?

Reply:
There’s no type for this object and the field is actually the dimension field. I didn’t think it was worth changing the interface since it is bard dimension specific